components/dashboards-web-component/src/utils/DashboardReportGenerator.js

```diff
@@ -307,6 +315,13 @@ export default class DashboardReportGenerator {
                             'footer', orientation, reportName, true);
                     });
                 }
+            } else {
+                const defaultHeaderImage = 'stream-processor.png';
```

Comment:
Don't hard-code default header image here. It should be in the deployement.yaml file.